client-web/src/exlibris-app/book-grid.html

```diff
@@ -0,0 +1,188 @@
+                color: white;
+                font-size: 12px;
+                text-align: center;
+                margin-top: 5px;
+            }
+
+            .bookContent > .synopsis > .expected {
+                background: var(--app-primary-color);
+                color: white;
+                font-size: 12px;
+                text-align: center;
+                margin-top: 5px;
+            }
+
+            .bookContent > .synopsis > .borrowed {
+                background: var(--paper-green-400);
+                color: white;
+                font-size: 12px;
+                text-align: center;
+                margin-top: 5px;
+            }
+
+            .bookContent > .synopsis > .should-return-soon {
+                background: var(--paper-green-400);
+                color: white;
+                font-size: 12px;
+                text-align: center;
+                margin-top: 5px;
+            }
+
+            .bookContent > .synopsis > .reserved {
+                background: var(--app-primary-color);
+                color: white;
+                font-size: 12px;
+                text-align: center;
+                margin-top: 5px;
+            }
+        </style>
+
+        <iron-list items="[[books]]" as="book" scroll-target="document" grid>
+            <template>
+                <div class="bookContainer">
+                    <div class="bookContent" tabindex$="[[tabIndex]]">
+                        <iron-image sizing="cover"
+                                    src="[[book.image]]">
+                        </iron-image>
+                        <div class="synopsis">
+                            <div class="title" title="[[book.title]]">[[book.title]]</div>
+                            <div class="author" title="[[book.author]]">[[book.author]]</div>
+                            <div class$="[[book.status]]">[[book.statusMessage]]</div>
+                        </div>
+                    </div>
+                </div>
+            </template>
+        </iron-list>
+
+        <!-- this element loads more books when the user scrolls down and reached the lower threshold -->
+        <iron-scroll-threshold id="scrollTheshold"
+                               lower-threshold="5"
+                               on-lower-threshold="_onLowerThreshold"
+                               scroll-target="document">
+        </iron-scroll-threshold>
+
+    </template>
+
+    <script>
+        /**
+         * `<book-grid>` components displays grid of books with picture, synopsis and status.
+         *
+         * @customElement
+         * @polymer
+         */
+        class BookGrid extends Polymer.Element {
+            static get is() {
+                return 'book-grid';
+            }
+
+            static get properties() {
+                return {
+                    /**
+                     * The array of books to display.
+                     */
+                    books: Array
```

Comment:
Please describe the object of books (fields of book object). 
See https://stackoverflow.com/a/31573441/3860844